### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by akashksinghal**
Added Graphs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,35 +4,42 @@ This is a library of various algorithms that are used in competitive programming
...
 
 ### Sorting algorithm 
-[Merge Sort](Algorithms/Sorting/merge_sort.cpp) <br/>
-[Heap Sort](Algorithms/Sorting/heap_sort.cpp)<br/>
-[Insertion Sort](Algorithms/Sorting/insertion_sort.cpp) <br/>
-[Bubble Sort](Algorithms/Sorting/bubble_sort.cpp) <br/>
-[Selection Sort](Algorithms/Sorting/selection_sort.cpp)
+* [Merge Sort](Algorithms/Sorting/merge_sort.cpp) <br/>
+* [Heap Sort](Algorithms/Sorting/heap_sort.cpp)<br/>
+* [Insertion Sort](Algorithms/Sorting/insertion_sort.cpp) <br/>*
```
Comment (Bhavnaharitsa) [inline README.md]: remove asterix here

**Review by akashksinghal — COMMENTED** [review]
Updated

File: README.md (hunk 1 above)
Comment (akashksinghal) [reply README.md]: Done

**Comment by akashksinghal** [comment]
I have resolved the conflicts as well.

**Comment by akashksinghal** [comment]
@Bhavnaharitsa Pleave review this PR

**Pull request merged**